### DISCUSSION TITLE
SC 3.3.2: Visible label is needed for Phone Number field?

### DISCUSSION THREAD

**Discussion by AccessibilityEnthusiastic** [w3c/wcag#2944] [Q&A]
Hi Everyone,
in this scenario(please look into the screenshot), when we select the country from Listbox automatically, the country code is also populated
 so how a user would know, whether they have to enter the mobile number or not

because if we click on continue then only error messages comes up
![Screenshot_5](https://user-images.githubusercontent.com/115781693/213474956-e3592868-5ee2-46b7-9c81-d88807b178ca.jpg)

=========================================================================
in this scenario(please look into the screenshot), when the user erases the country code, the sample mobile number and country code are visible as a placeholder text

and when we try to enter mobile number, the first entered two characters are automatically taken as the country code.
how user would know user has to provide country code along with the mobile number?

![Screenshot_6](https://user-images.githubusercontent.com/115781693/213476067-7d869226-ac3c-447b-a350-8364ff1496e1.jpg)

So my question is , is a visible label required here for the input field?

**Comment by JAWS-test** [comment]
I think three things are needed here:
* a visible and programmatic label (e.g. "phone number").
* an input hint, in which form the input has to be done. It is better to display the input hint permanently next to the field and not as a placeholder in the field (placeholder often has insufficient contrast, is not visible once there is text in the field, etc.).
* error message and further hints about the correct format and explanation why the current input is not correct, if an incorrect input has been made.
